src/Component/Rule/Rule.tsx

```diff
@@ -18,30 +18,46 @@ import { Data as GsData } from 'geostyler-data';
 import RuleNameField, { DefaultNameFieldProps } from '../NameField/NameField';
 import ComparisonFilterUi, { DefaultComparisonFilterProps } from '../Filter/ComparisonFilter/ComparisonFilter';
 // import RuleRemoveButton from './RemoveButton/RemoveButton';
-import ScaleDenominator, { DefaultScaleDenominatorProps } from '../ScaleDenominator/ScaleDenominator';
+import ScaleDenominator from '../ScaleDenominator/ScaleDenominator';
 import Fieldset from '../FieldSet/FieldSet';
 import Preview, { DefaultPreviewProps } from '../Symbolizer/Preview/Preview';
 
 import './Rule.css';
 
+import { localize } from '../LocaleWrapper/LocaleWrapper';
+
 import {
   cloneDeep as _cloneDeep
 } from 'lodash';
 
+// i18n
+export interface RuleLocale {
+    removeRuleBtnText: string;
+    scaleFieldSetTitle: string;
+    filterFieldSetTitle: string;
```

Comment:
I think the `Set` is not necessary in `scaleFieldSetTitle` and `filterFieldSetTitle`